.pre-commit-config.yaml

```diff
@@ -18,7 +18,6 @@ repos:
       rev: 3.9.0
       hooks:
           - id: flake8
-            exclude: run-tests/, src/parser/generated_parser.py
```

Comment:
I tried this and failed 😄

Reply:
ah yeah it's because .flake8 needs the `run-tests/*` whereas `.pre-commit-config` doesn't. I've gone back and forward several times trying to work that out